pkg/apis/metrics/v1alpha1/appmetricsapp_types.go

```diff
@@ -0,0 +1,45 @@
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// AppMetricsAppSpec defines the desired state of AppMetricsApp
+// +k8s:openapi-gen=true
+type AppMetricsAppSpec struct {
+	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
+	// Important: Run "operator-sdk generate k8s" to regenerate code after modifying this file
+	// Add custom validation using kubebuilder tags: https://book.kubebuilder.io/beyond_basics/generating_crd.html
+	Name string `json:"app-name"`
```

Comment:
Yes, we had a separate appName defined in the spec, the reason for this was that it could be updated in the MDC, which (I believe) metadata.name is not updateable. There was a discussion about removing the ability to update the app name in the MDC which I believe we are doing now so that may remove the need for this additional spec. But we should check that with @ziccardi or @craicoverflow before removing it. For now I will update to the better syntax until we can get some guidance from the guys.

Reply:
@grdryn @laurafitzgerald Correct. We already removed the ability to update the app name in the current master.

Reply:
So we should not use spec.appName?

Reply:
@ziccardi

Reply:
app-name removed. Done